plugins/gradle/gradle-plugin/src/main/java/org/wildfly/swarm/plugin/gradle/AbstractThorntailPlugin.java

```diff
@@ -48,13 +54,40 @@ public AbstractThorntailPlugin(ToolingModelBuilderRegistry registry) {
 
     @Override
     public void apply(Project project) {
+        this.project = project;
         // Skip if the plugin has already been applied.
         if (project.getExtensions().findByType(ThorntailExtension.class) == null) {
             ThorntailExtension extension = new ThorntailExtension(project);
             project.getExtensions().add(THORNTAIL_EXTENSION, extension);
 
             // Register a model builder as well.
             registry.register(new ThorntailToolingModelBuilder());
+
+            //noinspection Convert2Lambda
+            project.afterEvaluate(new Action<Project>() {
+                @Override
+                public void execute(Project p) {
+                    addDependency("runtimeOnly", "io.thorntail:bootstrap:" + determinePluginVersion());
```

Comment:
I am trying to see if I can program against the JEE APIs only, in which case we need to have the `bootstrap` library [available as a dependency](https://github.com/thorntail/thorntail/blob/master/tools/src/main/java/org/wildfly/swarm/tools/DependencyManager.java#L76).

You will see this in the examples as well.